lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -428,7 +428,7 @@ sub _show {
 }
 
 sub job_next_previous_ajax ($self) {
-    my $main_job = $self->get_current_job;
+    return $self->reply->not_found unless my $main_job = $self->_get_current_job;
```

Comment:
Would you mind extending our unit tests to cover this?

Reply:
I consider this not necessary, it's just basic in-line safety checks which doesn't even touch coverage. Please also see https://github.com/os-autoinst/openQA/pull/4575 where I try to make the overall code simpler with exception handling so that we don't need these checks at all.

Reply:
I don't follow. I'm asking if we can have a regression test since this issue was only seen in production.